{
  "id": "USN-6674-1",
  "summary": "python-django vulnerability",
  "details": "Seokchan Yoon discovered that the Django Truncator function incorrectly\nhandled very long HTML input. A remote attacker could possibly use this\nissue to cause Django to consume resources, leading to a denial of service.\n",
  "aliases": [],
  "related": [
    "CVE-2024-27351"
  ],
  "published": "2024-03-04T15:04:24.552253Z",
  "modified": "2024-03-04T15:04:24.552253Z",
  "affected": [
    {
      "package": {
        "ecosystem": "Ubuntu:20.04:LTS",
        "name": "python-django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2:2.2.12-1ubuntu0.22"
            }
          ]
        }
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-django-doc": "2:2.2.12-1ubuntu0.22",
            "python3-django": "2:2.2.12-1ubuntu0.22"
          }
        ],
        "availability": "No subscription needed"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:22.04:LTS",
        "name": "python-django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "2:3.2.12-2ubuntu1.11"
            }
          ]
        }
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-django-doc": "2:3.2.12-2ubuntu1.11",
            "python3-django": "2:3.2.12-2ubuntu1.11"
          }
        ],
        "availability": "No subscription needed"
      }
    },
    {
      "package": {
        "ecosystem": "Ubuntu:23.10",
        "name": "python-django"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "3:4.2.4-1ubuntu2.2"
            }
          ]
        }
      ],
      "ecosystem_specific": {
        "binaries": [
          {
            "python-django-doc": "3:4.2.4-1ubuntu2.2",
            "python3-django": "3:4.2.4-1ubuntu2.2"
          }
        ],
        "availability": "No subscription needed"
      }
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://ubuntu.com/security/notices/USN-6674-1"
    },
    {
      "type": "REPORT",
      "url": "https://ubuntu.com/security/CVE-2024-27351"
    }
  ]
}